Instagram/View Controllers/ComposeViewController.m

```diff
@@ -52,17 +51,14 @@ - (void) takePhoto:(BOOL)canTakePhoto {
 }
    //method from codepath
 - (void)imagePickerController:(UIImagePickerController *)picker didFinishPickingMediaWithInfo:(NSDictionary<NSString *,id> *)info {
-    
-    // Get the image captured by the UIImagePickerController
     UIImage *originalImage = info[UIImagePickerControllerOriginalImage];
     UIImage *editedImage = info[UIImagePickerControllerEditedImage];
-
+    
```

Comment:
Interesting enough, this is replacing a newline with whitespace.

Reply:
Make sure you have automatic whitespace trimming on if you don't already. 
![image](https://user-images.githubusercontent.com/4922820/125124868-a1982980-e0ad-11eb-90b5-d6cb3e2ced10.png)